build/depends.py

```diff
@@ -409,11 +446,33 @@ def configure(self, build, conf):
                 build.env.Append(LIBS = 'qico')
                 build.env.Append(LIBS = 'qsvg')
                 build.env.Append(LIBS = 'qtga')
+                build.env.Append(LIBS = 'qgif')
+                build.env.Append(LIBS = 'qjpeg')
+
+                # accessibility plugins (gone in Qt5)
+                if not qt5:
+                    build.env.Append(LIBPATH=[
+                        os.path.join(build.env['QTDIR'],'plugins/accessible')])
+                    build.env.Append(LIBS = 'qtaccessiblewidgets')
+
+                # platform plugins (new in Qt5 for Windows)
+                if qt5:
+                    build.env.Append(LIBPATH=[
+                        os.path.join(build.env['QTDIR'],'plugins/platforms')])
+                    build.env.Append(LIBS = 'qwindows')
+
+                # styles (new in Qt5 for Windows)
+                if qt5:
+                    build.env.Append(LIBPATH=[
+                        os.path.join(build.env['QTDIR'],'plugins/styles')])
+                    build.env.Append(LIBS = 'qwindowsvistastyle')
+
+                # sqldrivers (new in Qt5? or did we just start enabling them)
```

Comment:
They existed in Qt4 but we disabled them because we link directly with SQLite so it includes our build flags/enhancements.

Reply:
Note, this is a static plugin, not a DLL like the `qt_sqlite_plugin` scons parameter controls. I believe in Qt 4 static builds, the QtSql SQLite driver is built-in to Qt, and in Qt 5 when static built it has been moved to a static sqldriver plugin. Could be wrong...